AUTHORS.md

```diff
@@ -0,0 +1,45 @@
+# Credits
+
+---
+© 2024 **xdem developers**.
+
+**xdem** is licensed under permissive Apache 2 license (See LICENSE file).
+
+All contributors listed in this document are part of the **xdem developers**, and their
+contributions are subject to the project's copyright under the terms of the
+[Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0).
+
+See CONTRIBUTING.md for more details on Contributor License Agreement.
+
+This file keeps track of authors contributions.
+
+## Development Lead
+
+---
+
+- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <hugonnet@uw.edu>
```

Comment:
Is it common to have the emails of the main developers in this file? Just out of curiosity. If yes, OK for me!
We can replace by my public GitHub email instead: romain.hugonnet@gmail.com (as our institutional emails usually become deprecated after a couple years in early research years, moving all the time :sweat_smile: !)

Reply:
Amaury's email should remain the same for a while though!

Reply:
I don't know if it is common, but we have done that to be able to find all contributors if one day there is a need to contact all contributors on a global projet matter (communication, licence, global info...). So I think it is a good habit to have.